backend/scripts/seed_data.py

```diff
@@ -165,51 +167,89 @@
         test_results=[
             C3TestResult(
                 id=1,
-                name="test1",
-                status=C3TestResultStatus.FAIL,
-                category="",
-                comment="",
-                io_log="",
-            ),
-            C3TestResult(
-                id=2,
-                name="test2",
+                name="docker/compose-and-basic_armhf",
                 status=C3TestResultStatus.PASS,
-                category="",
+                category="Docker containers",
                 comment="",
-                io_log="",
+                io_log="""
+test Pulling 
+ test Pulled 
+ Network 4506_default  Creating
+ Network 4506_default  Created
+ Container 4506-test-1  Creating
+ Container 4506-test-1  Created
+ Container 4506-test-1  Starting
+ Container 4506-test-1  Started
+ Container 4506-test-1  Stopping
+ Container 4506-test-1  Stopped
+ Container 4506-test-3  Creating
+ Container 4506-test-2  Creating
+ Container 4506-test-1  Recreate
+ Container 4506-test-3  Created
+ Container 4506-test-2  Created
+ Container 4506-test-1  Recreated
+ Container 4506-test-1  Starting
+ Container 4506-test-1  Started
+ Container 4506-test-2  Starting
+ Container 4506-test-2  Started
+ Container 4506-test-3  Starting
+ Container 4506-test-3  Started
+3
+ Container 4506-test-1  Running
+ Container 4506-test-1  Stopping
+ Container 4506-test-1  Stopping
+ Container 4506-test-1  Stopped
+ Container 4506-test-1  Removing
+ Container 4506-test-1  Removed
+ Network 4506_default  Removing
+ Network 4506_default  Removed
+""",
```

Comment:
Nadzeya pointed out the textwrap module that has useful [dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) function in another PR a few days ago. Maybe we can apply it here to make the text indent in the same way as the object arguments.

Reply:
Oh nice find, will use it